File: src/test/java/test/GetUserLoggedInAppsTest.java
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class GetUserLoggedinAppsTest {

    private static final String ACCESS_KEY_ID = "YOUR_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "YOUR_ACCESS_KEY_SECRET";


    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions(ACCESS_KEY_ID, ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        GetUserLoggedinAppsDto request = new GetUserLoggedinAppsDto();
        request.setUserId("userId_6776");
        request.setUserIdType("userIdType_2096");

        UserLoggedInAppsListRespDto response = client.getUserLoggedinApps(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/GetUserLoggedinAppsTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.GetUserLoggedinAppsDto{
        UserId: "userId_2850",
        UserIdType: "userIdType_3338",
    }
    response := client.GetUserLoggedinApps(&request)
    fmt.Println(response)
}

